extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerDeploymentRecorder.java

```diff
@@ -60,6 +60,8 @@ private void initTracerConfig(JaegerConfig jaeger) {
         initTracerProperty("JAEGER_TAGS", jaeger.tags, tags -> tags.toString());
         initTracerProperty("JAEGER_PROPAGATION", jaeger.propagation, format -> format.toString());
         initTracerProperty("JAEGER_SENDER_FACTORY", jaeger.senderFactory, sender -> sender);
+        initTracerProperty(QuarkusJaegerTracer.LOG_TRACE_CONTEXT, jaeger.logTraceContext,
```

Comment:
Why is this property the only one defined  in a constant? I think it should be an all-or-nothing here :)

Reply:
I think it's a constant because it's used twice, once to be set and another time to be read. It makes mostly sense to me in this situation (and I'm the consistency maniac ;)).

I added a small comment about this below, though.

Reply:
The constant is used because it is used two times.